src/components/SEP/SEPPage.tsx

```diff
@@ -49,7 +49,7 @@ const SEPPage: React.FC = () => {
   }
 
   return (
-    <StyledContainer maxWidth={'xl'}>
+    <StyledContainer maxWidth='xl'>
```

Comment:
Isn't this creating an eslint error because usually we use doublequotes for attribute values.